app/api/game.yaml

```diff
@@ -0,0 +1,163 @@
+swagger: '2.0'
+info:
+  version: 0.1.0
+  title: 'Torwolf Games API'
+paths:
+  /games:
+    get:
+      description: Finds games
+      parameters:
+        - name: offset
+          in: query
+          description: The offset at which the result set should begin.
+          required: false
+          type: integer
+          default: 0
+        - name: limit
+          in: query
+          description: Result will contain at most limit items
```

Comment:
Haha, I agree. I think it makes sense for our documentation to be as polished as possible